go.mod

```diff
@@ -121,3 +124,21 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+//Here until we can drop dependency on github.com/jonathan-innis/aws-sdk-go-prometheus
+replace (
```

Comment:
Why do we need a replace? Can we directly just pin to these versions above? Any downstream integrations that rely on this package will use the wrong version if you don't directly pin and instead use replace statements to work around failed builds.